File: AnuitexTraining.BusinessLogicLayer/Models/PrintingEditions/PrintingEditionModel.cs
using System.Collections.Generic;
using AnuitexTraining.BusinessLogicLayer.Models.Base;
using static AnuitexTraining.Shared.Enums.Enums;

namespace AnuitexTraining.BusinessLogicLayer.Models.PrintingEditions
{
    public class PrintingEditionModel : BaseModel
    {
        public string Title { get; set; }
        public string Description { get; set; }
        public double Price { get; set; }
        public PrintingEditionType Type { get; set; }
        public CurrencyType Currency { get; set; }
        public List<string> Authors { get; set; }
    }
}

File: AnuitexTraining/ClientApp/src/app/modules/administrator/models/printing-edition.model.ts
import {PrintingEditionTypeEnum} from "../../shared/enums/printing-edition-type.enum";
import {CurrencyTypeEnum} from "../../shared/enums/currency-type.enum";

export interface PrintingEditionModel {
  id: number,
  title: string,
  description: string,
  type: PrintingEditionTypeEnum,
  authors: string[],
  price: number,
  currency: CurrencyTypeEnum
}
